packages/core/jkl/_spacing.scss

```diff
@@ -5,119 +5,132 @@
 
 /// Tilsvarer 2px
 /// @type Number
-$spacing-3xs: 0.125rem;
+$spacing-2: 0.125rem;
+$spacing-3xs: $spacing-2;
```

Comment:
Inkluderte dem ikke i kommentaren min på #3249 for å holde den kort, men ønsker meg litt mer SassDoc på t-skjortestørrelsene for å hjelpe VS Code. Variablene vil bli vist med strikethrough i IDE som støtter det (VS Code med Some Sass, f. eks.).

```
/// Tilsvarer 2px
/// @deprecated Bruk $spacing-2.
/// @type Number
```